### DISCUSSION TITLE
Appium 1.22.2 - the timeoutMs argument in installApp mobile command- is there any impact of this argument on simulator?

### DISCUSSION THREAD

**Discussion by svermagit** [appium/appium#16702] [Q&A] [answered]
Appium 1.22.2 - the timeoutMs argument in installApp mobile command- is there any impact of this argument on simulator.

I have tried to execute the tests multiple times by passing different values but no impact I have seen.

("installApp", ImmutableMap.of("app", <appPath>, "timeoutMs", 2));

Can I get some help on this.

**Comment by jlipps** [comment]
I'm not sure. Based on the code paths I've followed in looking at this, it doesn't look like [the simulator's installApp method](https://github.com/appium/appium-ios-simulator/blob/1099c41102a35fec57dd4255085354ea9c60dd9f/lib/simulator-xcode-6.js#L196) does anything with a timeout. @mykola-mokhnach is `mobile: installApp` supposed to do something with this timeout value?

**Comment by mykola-mokhnach** [comment, marked as answer]
No, on simulator this arg does not have any effect, only real devices

**Reply by svermagit** [reply]
Thank you for the quick response @mykola-mokhnach and @jlipps
